internal/utils.go

```diff
@@ -0,0 +1,28 @@
+package internal
+
+import (
+	"vulnbot/logger"
+
+	"github.com/spf13/pflag"
+)
+
+func checkErr(err error) {
+	log := logger.Get()
+	if err != nil {
+		log.Info().Err(err).Msg("Unexpected error extracting the flag.")
```

Comment:
[q] What types of errors would be encountered here? Would this be like if somebody provided the wrong data type to a flag?

Reply:
That wouldn't happen because `cobra` would take care of that:

```
jose@Joses-MacBook-Pro pinwheel/vulnbot (feature/add-flags-with-cobra-cli) » go run main.go scan --da
Error: unknown flag: --da
```

To be honest, I realize that the error message needs to be revised. I originally included the function to specifically handle errors when extracting the flag. However, now that I'm thinking, I realized that if someone were to provide an incorrect flag name, such as `getString(cmd.Flags(), "WRONG_NAME")`, it could potentially result in an error within the function.

Reply:
Can we have an epic enforcing and adding tests to the project? 😅

Reply:
GitHub doesn't have epics 😄 But I am planning to implement codecov.io to measure and enforce testing coverage.

Reply:
Hmmmm so reading up on these `getBool`, etc, I think this should actually be a `log.Fatal()`.

> There are helper functions available to get the value stored in a Flag if you have a FlagSet but find it difficult to keep up with all of the pointers in your code. If you have a pflag.FlagSet with a flag called 'flagname' of type int you can use GetInt() to get the int value. But notice that 'flagname' must exist and it must be an int. GetString("flagname") will fail.

So that says, to me, that these errors should only ever occur during development. And we should absolutely not continue execution if they occur.

Reply:
Yup! I agree with you that this should be considered a terminal error. Will update!

Reply:
Fixed!